JavaScript homeworks/homework-6.js

```diff
@@ -0,0 +1,193 @@
+//Loops task 1//
+let PersonDescription = ['name: John', 'age: 30', 'city: New-York'];
```

Comment:
this was supposed to be an object, not array

Reply:
Added changes, thank you!